Tests/MongoSwiftTests/AsyncAwaitTestUtils.swift

```diff
@@ -165,9 +165,9 @@ extension MongoClient {
     }
 
     internal func getUnmetRequirement(_ testRequirement: TestRequirement) async throws -> UnmetRequirement? {
-        async let topologyType = try self.topologyType()
-        async let serverVersion = try self.serverVersion()
-        async let params = try self.serverParameters()
+        async let topologyType = try await self.topologyType()
```

Comment:
happened to notice while adding new test using this that this method was actually still calling the blocking methods that `wait()` to get these values. oops